Dockerfile

```diff
@@ -1,7 +1,7 @@
 
 # --platform=$BUILDPLATFORM is used build javascript source with host arch
 # Otherwise TS builds on emulated archs and can be extremely slow (+1h)
-FROM --platform=${BUILDPLATFORM:-amd64} node:18-alpine as build_src
+FROM --platform=${BUILDPLATFORM:-amd64} node:20-alpine as build_src
```

Comment:
This can lead to non-deterministic builds, we might install a different node version on the release compared to the rc.

Would suggest we at least specify the minor version to reduce this risk with the drawback that we have to more frequently update the value here.

```suggestion
FROM --platform=${BUILDPLATFORM:-amd64} node:20.4-alpine as build_src
```

Reply:
Lets save this for another PR and deeper discussion